packages/harpstrata/src/get-harp-strata/get-harp-strata.ts

```diff
@@ -9,26 +14,38 @@ export const getHarpStrata = (props: HarpStrataProps): HarpStrata => {
   const { tuningId, valvingId, pozitionId, harpKeyId, activeIds } = props
 
   const apparatus = buildApparatus(tuningId, valvingId)
+
   const { rootOffset } = getPozition(pozitionId)
   const pitch = getPitch(harpKeyId)
+  const { id: pitchId } = pitch
 
   const { halfstepIndexMatrix } = apparatus
-  const { id: pitchId } = pitch
+  const degreeMatrices = {
```

Comment:
This could all be much nicer with a higher order function. That higher order function might require that the parameters that the function it will call takes be a single property so that type checking can be applied to it. Otherwise we'll have to give it a non-specific number of arguments to pass on to the function `getDegreeMatrix` in this case.

Reply:
This has been handled in #130, but I'm going to leave this comment open as a way to remind that additional tests are required anywhere where the `mapHarpFaceFacts` function was used, to account for the possibility that a chromatic config could pass through the function.

Reply:
The only places which don't have additional tests now are `getHarpStrata` & `buildApparatus`.

Reply:
Raised as techdebt: https://github.com/js-jslog/harpguru/issues/131#issuecomment-985442182